apps/src/panels/panels.module.scss

```diff
@@ -45,6 +46,13 @@
       padding: 20px;
       animation: fade-in 1.5s;
 
+      h1, h2, h3, h4 {
```

Comment:
Just flagging this for @levadadenys - wondering if we should include our typography styles in our existing markdown components

Reply:
@sanchitmalhotra126 @breville   Good question,
It depends on designs you have for screens you develop. From what I see here in styles - these h1-h4 will look different from what we have in typography therefore it's ok not to use typography.

One nit though(for future probably since it's already merged): for design system we decided to use `rem` units for sizings where possible instead of `px` and `em` (for example for font-size, marging, padding properties, etc)